File: project_euler/010 - Summation of primes/10.java
class A {

    static int i, a;
    static long sum = 0;

    public static void main(String[] args) {
        for (int n = 3; n < 2000000; n += 2) {

            for (i = 3; i < n; i += 2) {
                if (n % i == 0) {
                    System.out.println("Not a prime number..................... " + n);
                    break;
                }
            }
            if (i == n) {
                System.out.println("Prime Number " + n);
                sum += n;
            }
        }
        System.out.println("Sum : "+(sum+2));
    }
}

File: project_euler/010 - Summation of primes/10.py
import math

def is_prime(n):
    if n == 2:
        return True
    else:
        sqr = int(math.sqrt(n)) + 1
        for divisor in range(2, sqr):
            if n % divisor == 0:
                return False
    return True


prime = [2]
i = 3

for i in range(3, 2000000, 2):
    if is_prime(i):
        prime.append(i)

mysum = sum(prime)

print(f"The sum of all the primes below two million: {mysum}")
